docs/usage/self-hosted-experimental.md

```diff
@@ -39,6 +39,10 @@ You can set the config file Renovate should read with the `RENOVATE_CONFIG_FILE`
 
 The process that runs Renovate must have the correct permissions to delete the config file.
 
+## `RENOVATE_X_DISABLE_DOCKER_HUB_LABELS`
+
+If set to any value, Renovate will skip attempting to get release labels (e.g. gitRef, sourceUrl) from manifest annotations for `https://index.docker.io`.
```

Comment:
add a more descriptive text about the side effects. eg no changelogs because of no source url.

Reply:
I've added additional notes to the docs about side effects setting this variable may result in.